### ISSUE TITLE
Roadmap for GraphQL 15

### ISSUE THREAD

**Issue by iamchathu** [MichalLytek/type-graphql#594]
**Is your feature request related to a problem? Please describe.**
Any update for GraphQL version 15 support. Currently give error for not supported GraphQL version

```bash
UnhandledPromiseRejectionWarning: Error: Looks like you use an incorrect version of the 'graphql' package: "15.0.0". Please ensure that you have installed a version that meets TypeGraphQL's requirement: "^14.1.1".
```

**Describe the solution you'd like**
Update on Roadmap towards GraphQL new version.

**Comment by MichalLytek** [comment]
I need to go through all the breaking changes and fixes, adjust the code appropriately and fix all the encountered issued:
https://github.com/graphql/graphql-js/releases

Don't expect this to be backported to `0.17.x` branch - this will be a part of a major `v1.0.0` release 😉

**Comment by iamchathu** [comment]
Seems there is lot breaking changes and understand it will take time to co with it. But love to see it added to 1.0 release. Seems my Lerna setup added it as project dependancy. After removing it the error is gone.

**Comment by MichalLytek** [comment]
The first issue I've encountered is this bug:
https://github.com/graphql/graphql-js/issues/2504

Based on the experience with `v14`, it gonna takes a few weeks or longer to update all other libraries like apollo libraries to work with v15. For now it's 10+ peer dependencies error.

**Comment by MohammedALREAI** [comment]
try to remove  graphql  by yarn remove  graphql and install it again with version      "graphql": "^14.1.1",

**Comment by MichalLytek** [comment]
As no response from `graphql-js` maintainers, I gonna move forward and re-enable that failing test case later, when they release a patch for that.

So closing via a67c3af 🔒 Available in `1.0.0-rc.1` release 😉
